DatadogWebViewTracking/Sources/WebViewTracking.swift

```diff
@@ -106,15 +140,38 @@ public enum WebViewTracking {
             .map { return "\"\($0)\"" }
             .joined(separator: ",")
 
+        let privacyLevel = sessionReplayConfiguration?.privacyLevel ?? .mask
+
+        // Share native capabilities with Browser SDK
+        let capabilities: String = {
+            var capabilities: [String] = []
+
+            // Add 'records' capability when session-replay
+            // is configured.
+            if sessionReplayConfiguration != nil {
+                capabilities.append("records")
+            }
+
+            return capabilities
+                .map { return "\"\($0)\"" }
+                .joined(separator: ",")
+        }()
```

Comment:
suggestion/ Unless we anticipate more capabilities in short-term, it can be as simple as:
```swift
let capabilities = sessionReplayConfiguration != nil ? "\"records\"" : ""
```

Reply:
Indeed, this interface is meant to support multiple capabilities but not in short term, I will change this.